internal/e2e/scoring_test.go

```diff
@@ -79,8 +79,8 @@ func TestBM25(t *testing.T) {
 			query:    &query.Substring{Pattern: "example"},
 			content:  exampleJava,
 			language: "Java",
-			// bm25-score: 0.58 <- sum-termFrequencyScore: 14.00, length-ratio: 1.00
```

Comment:
I remove the score from the comments because it is redundant and requires an update for every update of the scoring function. sum-termFrequencyScore and length-ratio are more robust.